### PR TITLE
[16.0][ADD] partner_product_price

### PR DESCRIPTION
**Pull request by norlinhenrik**
- Open a contact.
- Click Actions - Products.
- In list view, show the fields **Contact** and **Contact's Price**.

### CODE REVIEW

File: partner_product_price/data/ir_actions_server.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="action_product_price_list_report" model="ir.actions.server">
+        <field name="name">Products</field>
+        <field name="groups_id" eval="[(4, ref('resource_booking.group_user'))]" />
```
Comment (legalsylvain) [inline partner_product_price/data/ir_actions_server.xml:5]: this module ``resource_booking`` is not set in the dependency field of the manifest.

File: partner_product_price/models/product_product.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                )
+
+    partner_id = fields.Many2one(
+        "res.partner",
+        string="Contact",
+        compute="_compute_partner_and_price",
+    )
+    partner_price = fields.Float(
+        "Contact's Price",
+        compute="_compute_partner_and_price",
```
Comment (legalsylvain) [inline partner_product_price/models/product_product.py:29]: could you add precision ?

File: partner_product_price/data/ir_actions_server.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="action_product_price_list_report" model="ir.actions.server">
+        <field name="name">Products</field>
```
Comment (legalsylvain) [inline partner_product_price/data/ir_actions_server.xml]: should be a more explicit name, don't you think ? something like 'See product prices"

File: partner_product_price/data/ir_actions_server.xml (hunk 1 above)
Comment (legalsylvain) [inline partner_product_price/data/ir_actions_server.xml]: this module resource_booking is not set in the dependency field of the manifest.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
thanks for your contribution !  
Some blocking point in line.
could you take a look on pre-commit error also ? 

thanks !

**Comment by norlinhenrik** [comment]
@legalsylvain Thank you for the input! See the second commit with fixes plus more.

**Comment by norlinhenrik** [comment]
Do you think that this module should be here in `product-attribute` or in `partner-contact`? I put it here because since other modules related to pricing are here.

File: partner_product_price/models/product_product.py (hunk 2 above)
Comment (norlinhenrik) [reply partner_product_price/models/product_product.py:29]: Done

File: partner_product_price/data/ir_actions_server.xml (hunk 3 above)
Comment (norlinhenrik) [reply partner_product_price/data/ir_actions_server.xml]: Done.

File: partner_product_price/data/ir_actions_server.xml (hunk 1 above)
Comment (norlinhenrik) [reply partner_product_price/data/ir_actions_server.xml]: The reference to resource_booking is removed.

**Pull request closed without merging**